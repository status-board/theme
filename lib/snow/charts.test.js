"use strict";

var _charts = _interopRequireDefault(require("./charts"));

function _interopRequireDefault(obj) { return obj && obj.__esModule ? obj : { "default": obj }; }

test('Charts should match snapshot', function () {
  expect(_charts["default"]).toMatchSnapshot();
});
//# sourceMappingURL=data:application/json;charset=utf-8;base64,eyJ2ZXJzaW9uIjozLCJzb3VyY2VzIjpbIi4uLy4uL3NyYy9zbm93L2NoYXJ0cy50ZXN0LnRzIl0sIm5hbWVzIjpbInRlc3QiLCJleHBlY3QiLCJjaGFydHMiLCJ0b01hdGNoU25hcHNob3QiXSwibWFwcGluZ3MiOiI7O0FBQUE7Ozs7QUFFQUEsSUFBSSxDQUFDLDhCQUFELEVBQWlDLFlBQVk7QUFDN0NDLEVBQUFBLE1BQU0sQ0FBQ0Msa0JBQUQsQ0FBTixDQUFlQyxlQUFmO0FBQ0gsQ0FGRyxDQUFKIiwic291cmNlc0NvbnRlbnQiOlsiaW1wb3J0IGNoYXJ0cyBmcm9tICcuL2NoYXJ0cyc7XG5cbnRlc3QoJ0NoYXJ0cyBzaG91bGQgbWF0Y2ggc25hcHNob3QnLCAoKTogdm9pZCA9PiB7XG4gICAgZXhwZWN0KGNoYXJ0cykudG9NYXRjaFNuYXBzaG90KCk7XG59KTtcbiJdfQ==