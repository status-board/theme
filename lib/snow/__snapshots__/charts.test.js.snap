// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Charts should match snapshot 1`] = `
Object {
  "axis": Object {
    "legendColor": "#333",
    "textColor": "#333",
    "tickColor": "#bbb",
  },
  "colors": Array [
    "#82c2de",
    "#ade0de",
    "#dad7a3",
    "#b3deff",
    "#baab89",
  ],
  "grid": Object {
    "stroke": "#eee",
  },
  "tooltip": Object {
    "background": "#fff",
    "borderRadius": "2px",
    "boxShadow": "0 5px 9px rgba(0, 0, 0, 0.15)",
    "color": "#333",
    "fontSize": "1.4vmin",
  },
}
`;
